src/query/ts/m3db/convert.go

```diff
@@ -21,11 +21,14 @@
 package m3db
```

Comment:
can we have a high level doc of how our conversion is working ? We can put that in the user docs since that would help other storages to integrate with us if needed

Reply:
Will put one together. It'll be a bit more involved since it will cover how split blocks will work so may take a bit longer to put together unfortunately

Reply:
should we be coding before figuring out how to do it ? :P It was just a suggestion to reduce back n forth and avoiding having you to redo lot of things during reviews but i don't mind either way.

Reply:
Haven't started the other section yet, sho shouldn't be a problem there :)